.github/workflows/build_and_test.yml

```diff
@@ -686,6 +686,14 @@ jobs:
         fi
         cd docs
         bundle exec jekyll build
+        cd ..
+    - name: Tar documentation
+      run: tar cjf site.tar.bz2 docs/_site
+    - name: Upload documentation
+      uses: actions/upload-artifact@v3
```

Comment:
that depends on whether we need to monitor the build-doc in apache/repo.
I personally want to also upload in apache/repo, since the cost seems quietly small.

Reply:
on second thought, I think we can add this condition first, to be more conservative